etc/httpd/global.conf

```diff
@@ -126,10 +126,9 @@ RewriteRule ^/ja/?$ http://www.mozilla.jp/ [L,R]
 RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloads/ [L,R=301]
 
 # bug 1138280
-RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://www.mozilla.jp/$1/$2/notes/ [L,R=301]
```

Comment:
Ah, I understood what you mean.
I changed that rule as
    RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/? http://www.mozilla.jp/$1/$2notes/ [L,R=301]

It will work as same as you mentioned before. (by the way I found another typo in your suggestions).

Thanks